### DISCUSSION TITLE
Welcome to django-language-server Discussions!

### DISCUSSION THREAD

**Discussion by joshuadavidthomas** [joshuadavidthomas/django-language-server#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mouarius** [comment]
Hi everyone !
In my desperate quest to prove my colleagues working with PyCharm on a Django project, that I'm fine with Neovim, I feel like I'm slowly starting to run out of arguments to justify my choices...

This is a fact, **Django DX is way better in PyCharm** (dynamic type checking for models, navigation between templates like with an lsp, finding a view from its url, and more...).

Rather than leaving behind my fabulous Neovim configuration that i love working with, I've been starting to look at ways to improve my DX in Neovim with Django (I couldn't be the only Django dev working in Neovim, right guys ?). So in my long journey I've stumbled upon this repo and I'm so glad it exists ! I have to say that I'm not a LSP pro, neither I have never coded in Rust but I'm willing to contribute the best I can to this project.

I hope that this project hasn't been abandoned since, and I'm really looking forward to work on it !

I'm just installing everything and trying to make the one feature work on my config, and when I'll be more comfortable with it, I would be happy to give you some help !

Don't hesitate to contact me ofc, those are my first steps in the open source world as well !

Until next time, have a good one !

**Reply by joshuadavidthomas** [reply]
Thanks for your interest! Not abandoned, just on a momentary pause as the primary (and only at the moment, though I welcome any and all) author got sidetracked and swamped with the work that pays the bills. 😅

In all seriousness, the project is in the very early stages but should (hopefully) at installable! Especially if you're on Neovim, as that's my editor of choice too. Any sort of feedback about issues you've run into about getting things setup would certainly be valuable. 😄

There is also https://github.com/fourdigits/django-template-lsp which is in a much more complete state and more active at this point than this project has been recently.

**Reply by Mouarius** [reply]
I'm glad to see your response ! Happy to join !

I've managed to install the lsp server thanks to the nvim plugin in my configuration. It didn't work right away, I had to do two things for it to work, I did a second [discussion](https://github.com/joshuadavidthomas/django-language-server/discussions/75#discussion-8217344) for it to be more precise to installation on neovim, and be more discoverable than lost in the comments of presentations !
